src/RouteBinderServiceProvider.php

```diff
@@ -32,9 +45,10 @@ public function bootRoutes()
         /** @type \Illuminate\Routing\Router $router */
         $router = $this->app['router'];
 
-        foreach ($config->get('route-binder::routes') as $binder)
-        {
-            $this->app->make($binder)->bind($router);
+        if ($config->has('route-binder')) {
```

Comment:
The issue I have without this `if`-clause is that I get `Invalid argument supplied for foreach()` since the loop tries to run with data that isn't available in the config. Due to that exception I also can't run `php artisan vendor:publish` which results in manually publishing the config.